balrog.wsgi

```diff
@@ -42,3 +54,6 @@ application.config['SENTRY_PROCESSORS'] = ['auslib.util.sentry.SanitizeHeadersPr
 
 if application.config['SENTRY_DSN']:
     sentry.init_app(application)
+
+if newrelic:
+    application = newrelic.agent.wsgi_application()(application)
```

Comment:
My vote is no... we can, but I don't envision much value there. I don't foresee us ever being concerned about performance on the admin interface (usage being so much lower and limited to internal people/tools), whereas we already are for the update side.